### PR TITLE
chore(locale): update es and fr po

### PR DESCRIPTION
**Pull request by Bundy01**

### CODE REVIEW

File: locale/es.po [hunk 1]
```diff
@@ -531,14 +531,13 @@ msgstr "[c] git checkout -- '*'"
...
-#: pikaur/install_cli.py:208 pikaur/build.py:703
+#: pikaur/build.py:703 pikaur/install_cli.py:208
 msgid "[e] edit PKGBUILD"
 msgstr "[e] editar el PKGBUILD"
 
 #: pikaur/build.py:700
-#, fuzzy
 msgid "[f] check() skip"
-msgstr "[c] omisión de sumas de comprobación"
+msgstr "[f] verificación() ignorada"
```
Comment (actionless) [inline locale/es.po:540]: check() is name of funtion in PKGBUILD, not a word

File: locale/fr.po [hunk 2]
```diff
@@ -532,14 +532,13 @@ msgstr "[c] git checkout -- '*'"
...
-#: pikaur/install_cli.py:208 pikaur/build.py:703
+#: pikaur/build.py:703 pikaur/install_cli.py:208
 msgid "[e] edit PKGBUILD"
 msgstr "[e] éditer le PKGBUILD"
 
 #: pikaur/build.py:700
-#, fuzzy
 msgid "[f] check() skip"
-msgstr "[c] sauter les sommes de contrôle"
+msgstr "[f] vérification() ignorée"
```
Comment (actionless) [inline locale/fr.po:541]: same here

**Comment by actionless** [comment]
mb the original english string need better wording there, any suggestion are welcome

File: locale/fr.po (hunk 2 above)
Comment (Bundy01) [reply locale/fr.po:541]: Hi,
how should I translate this line? 
- do not change anything ("[f] check() skip")
- or "[f] ignorer check()"

Comment (actionless) [reply locale/fr.po:541]: i'm not sure if original english string is clear enough

Comment (Bundy01) [reply locale/fr.po:541]: Ok,
I'm waiting for your modification.

Comment (actionless) [reply locale/fr.po:541]: @Bundy01 what the output of ` makepkg --help | grep '\-\-nocheck'` in your languages?

Comment (Bundy01) [reply locale/fr.po:541]: @actionless: 
Ne pas exécuter la fonction check() dans PKGBUILD
Do not execute the check() function in PKGBUILD (translated from the original French message into English)

Comment (actionless) [reply locale/fr.po:541]: done!

**Comment by actionless** [comment]
thanks!

**Pull request merged**